app/frontend/Calendar/DisplayEvent/Invitation.svelte

```diff
@@ -14,88 +14,119 @@
+        <ButtonMenu buttonIcon={ChevronDownIcon}>
+          <MenuItem
+            label={$t`Change to Reject *=> not attend the meeting`}
+            onClick={onDecline}
+            icon={DeclineIcon}
+            classes="decline font-normal" />
+          <MenuItem
+            label={$t`Change to Maybe *=> Maybe attend the meeting`}
+            onClick={onTentative}
+            icon={MaybeIcon}
+            classes="maybe font-normal" />
+        </ButtonMenu>
+      {:else if myResponse == InvitationResponse.Decline}
+        <Button
+          label={$t`Rejected *=> A meeting request has been declined by you`}
           icon={DeclineIcon}
-          classes="decline font-normal" />
-        <MenuItem
-          label={$t`Change to Maybe *=> Maybe attend the meeting`}
-          onClick={onTentative}
-          icon={MaybeIcon}
-          classes="maybe font-normal" />
-      </ButtonMenu>
-    </hbox>
-  {:else if false}
-    <hbox class="buttons">
-      <Button
-        label={$t`Rejected *=> A meeting request has been declined by you`}
-        icon={DeclineIcon}
-        selected={true}
-        disabled={true}
-        classes="decline done font-normal" />
-      <ButtonMenu buttonIcon={ChevronDownIcon}>
-        <MenuItem
-          label={$t`Change to Accept *=> attend the meeting`}
+          selected={true}
+          disabled={true}
+          classes="decline done font-normal" />
+        <ButtonMenu buttonIcon={ChevronDownIcon}>
+          <MenuItem
+            label={$t`Change to Accept *=> attend the meeting`}
+            onClick={onAccept}
+            icon={AcceptIcon}
+            classes="accept font-normal" />
+          <MenuItem
+            label={$t`Change to Maybe *=> Maybe attend the meeting`}
+            onClick={onTentative}
+            icon={MaybeIcon}
+            classes="maybe font-normal" />
+        </ButtonMenu>
+      {:else}
+        <Button
+          label={$t`Confirm *=> Confirm to attend the meeting`}
           onClick={onAccept}
           icon={AcceptIcon}
           classes="accept font-normal" />
-        <MenuItem
-          label={$t`Change to Maybe *=> Maybe attend the meeting`}
-          onClick={onTentative}
-          icon={MaybeIcon}
-          classes="maybe font-normal" />
-      </ButtonMenu>
-    </hbox>
-  {/if}
+        <Button
+          label={$t`Reject *=> Decline to attend the meeting`}
+          onClick={onDecline}
+          icon={DeclineIcon}
+          classes="decline secondary font-normal" />
+        <ButtonMenu buttonIcon={ChevronDownIcon}>
+          <MenuItem
+            label={$t`Maybe *=> Maybe attend the meeting`}
+            onClick={onTentative}
+            icon={MaybeIcon}
+            classes="maybe font-normal" />
+        </ButtonMenu>
+      {/if}
+    {:else if message.invitationMessage}
+      <Button label={$t`Update calendar`} disabled={updateDisabled} onClick={onUpdate} classes="font-normal" />
```

Comment:
I don't understand what this button does. (And if I don't understand, how will a user understand?)

Reply:
Can you please do that automatically and remove the button?

Reply:
Good. Nice. Do that.

Reply:
Please do that in a new PR, so that this here can land.

Reply:
Filed #619